js/index.js

```diff
@@ -0,0 +1,122 @@
+    description:
+      ' bekler studies commons-based Berkman professor of Entreneurial Legal studies at harvard Law school',
+    photo: '/img/profiles/christopher-campbell-rDEOVtE7vOs-unsplash.jpg',
+    alt: 'Photo by Christopher Campbell on Unsplash',
+  },
+  {
+    name: 'Julina Hey',
+    proffession:
+      'Berkman professor of Entreneurial Legal studies at harvard Law school',
+    description:
+      ' bekler studies commons-based Berkman professor of Entreneurial Legal studies at harvard Law school',
+    photo: '/img/profiles/jack-finnigan-rriAI0nhcbc-unsplash.jpg',
+    alt: 'Photo by Jack Finnigan on Unsplash',
+  },
+  {
+    name: 'Julina Hey',
+    proffession:
+      'Berkman professor of Entreneurial Legal studies at harvard Law school',
+    description:
+      ' bekler studies commons-based Berkman professor of Entreneurial Legal studies at harvard Law school',
+    photo: '/img/profiles/tamarcus-brown-29pFbI_D1Sc-unsplash.jpg',
+    alt: 'Photo by Tamarcus Brown on Unsplash',
+  },
+];
+const openNav = () => {
+  document.querySelector('header').classList.add('visible');
+  document.querySelector('.closeBtn').classList.toggle('visible');
+};
+const closeMenu = () => {
+  document.querySelector('header').classList.remove('visible');
+  document.querySelector('.closeBtn').classList.toggle('visible');
+};
+document.querySelector('.fa-bars').addEventListener('click', openNav);
+document.querySelector('.closeBtn>i').addEventListener('click', closeMenu);
+document.querySelector('.nav-ul >li > a').addEventListener('click', closeMenu);
+const generateSpeaker = (index) => {
+  const container = document.createElement('div');
+  const image = document.createElement('div');
+  const chess = document.createElement('img');
+  const photo = document.createElement('img');
+  const fullname = document.createElement('h3');
+  const proffession = document.createElement('h4');
+  const description = document.createElement('p');
+  const information = document.createElement('div');
+  container.appendChild(image);
+  container.appendChild(information);
+  image.appendChild(chess);
+  image.appendChild(photo);
+  information.appendChild(fullname);
+  information.appendChild(proffession);
+  information.appendChild(description);
+  container.className = 'container';
+  image.className = 'big-image';
+  chess.className = 'chess-image';
+  photo.className = 'profile-image';
+  fullname.className = 'full-name';
+  proffession.className = 'profession';
+  description.className = 'description';
+  information.className = 'information';
+  document.querySelector('.speakers').appendChild(container);
+  fullname.innerText = speakers[index].name;
+  chess.src = './img/profiles/Chess_board.svg.png';
+  photo.src = speakers[index].photo;
+  photo.alt = speakers[index].alt;
+  proffession.innerText = speakers[index].proffession;
+  description.innerText = speakers[index].description;
+};
```

Comment:
Nice approach but you could do it better by iterating through the array of speakers and then displaying the data in their respective HTML tag.

An example of that could be this 

```
window.onload = () => {
  speakerList.map(speaker => {
    // code to build the new HTML element, it can use `createElement()` or just template strings(``).
    // it could attach the new element to a parent inside within the loop
    // or accumulate the generate elements in a variable and attach them all after it
  })
}

```

Reply:
thank you for your suggestion, I  will try to use `map()` instead of `for loop`. 
- I want to give you a general idea about My algorithm: 
"I want to generate speakers using their index in the array of the objects. In the approach that I used, I want to generate only 2 speakers in the mobile version and if the user clicks on more the rest speakers will be generated. whereas in the desktop version, all the speakers will be generated when the page is loaded. if there are any suggestions, I will be happy to get them"

Reply:
You can still achieve what you want to do with `map` and for you to display two speakers for mobile and when click on more you should display more speaker, basically all you need here is the `slice(starting index, endIndex)`, you can read more about how to use it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice)

Reply:
Thank you, @abruzy for your suggestions, I will use `slice()` function for that.